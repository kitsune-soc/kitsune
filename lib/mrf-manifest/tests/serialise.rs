#![cfg(feature = "serialise")]

const MANIFEST: &str = include_str!("test-manifest.json");

#[test]
fn encode_works() {
    let manifest = serde_json::from_str(MANIFEST).unwrap();
    let encoded_manifest = mrf_manifest::serialise(&manifest).unwrap();
    let encoded_manifest_str = String::from_utf8(encoded_manifest).unwrap();
    insta::assert_snapshot!(encoded_manifest_str);
}
