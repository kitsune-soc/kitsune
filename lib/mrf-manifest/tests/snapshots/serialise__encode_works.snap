---
source: lib/mrf-manifest/tests/serialise.rs
expression: encoded_manifest_str
---
{"activityTypes":["Announce","Create","Delete","Like"],"apiVersion":"v1","configSchema":null,"manifestVersion":"v1","name":"test-manifest","version":"0.0.1-cattywampus"}
